src/components/ComingSoon/ComingSoon.js

```diff
@@ -0,0 +1,48 @@
+import Button from '../Button/Button.vue';
+import Heading from '../Heading/Heading.vue';
+import TextParagraph from '../TextParagraph/TextParagraph.vue';
+import comingSoonImage from '../../assets/img_wip.svg';
+
+const components = {
+  Button,
+  Heading,
+  TextParagraph,
+};
+
+const data = function () {
```

Comment:
Please remove this unnecessary `data` block (it's no longer needed after adding `ComingSoonImage` to the `components` block, above.